### ISSUE TITLE
As an customer, I want to be identified by this app.

### ISSUE THREAD

**Issue by zhoumiabc** [zhoumiabc/Tofu-Kitchen-App#1]
For example, when I using this app, I want this app could remember my personal profile, so the restaurant could know who is ordering and where the ordering to deliver.
My personal opinion, Sign-up and Sign-in can help to achieve this.

**Comment by zhangda77** [comment]
I agree

**Comment by zhoumiabc** [comment]
Here is our app's prototype: http://qdgkw9.axshare.com/#p=home
I think it is better to have a sign in and sign up function that make the app remember the customers' information. 
I just try our prototype, and I think we need a go back button to let customer go back to previous menu when they get themselves into a wrong page.

**Comment by ghost** [comment]
I think we can also add a sign-out button for customers in case they use other people's cell phone to order and want to log out of there account when they complete ordering.

**Comment by zhoumiabc** [comment]
I think customers may want to input their name, address, or phone number when they sign up in order to let restaurant deliver their food in the right place.

**Comment by zhangda77** [comment]
It is so good that the system can identify where I am located so that I dont have to re-enter my personal information for the following orders. I can also find out the delivery time by using the App. Can I get a message if the delivery is postponed for some reasons?

**Comment by shabbirw** [comment]
Absolutely, because that is the point of our app. You only have to register once. After you register with your details, the restaurant will know where to deliver, how you make payment, etc.
